logproxy-filter-replace/main.go

```diff
@@ -1,49 +1,62 @@
 package main
 
 import (
-	"os"
+	"github.com/hashicorp/go-plugin"
+	"github.com/philips-software/logproxy/shared"
 	"regexp"
 	"strings"
 
 	"github.com/hashicorp/go-hclog"
-	"github.com/hashicorp/go-plugin"
 	"github.com/philips-software/go-hsdp-api/logging"
-	"github.com/philips-software/logproxy/shared"
 )
 
 var (
 	log = hclog.Default()
 )
 
-type TriggerReplace struct {
+type FilterReplace struct {
 	pattern *regexp.Regexp
 	replace string
 }
 
-func (f TriggerReplace) Filter(msg logging.Resource) (logging.Resource, bool, bool, error) {
+type Filter struct {
+	filterList []FilterReplace
+}
+
+func parse(config []Config) (ret []FilterReplace) {
+	for _, filterConfig := range config {
+		filter := &FilterReplace{}
+		compiled, err := regexp.Compile(filterConfig.Pattern)
+		if err != nil {
+			log.Error("Failed to compile regexp", "regexp", filterConfig.Pattern)
```

Comment:
Maybe document in README that invalid regexes will be skipped.

Reply:
Ahh cool, nice one!